Sources/Common/Communication/EventStorage.swift

```diff
@@ -79,7 +79,8 @@ actor EventStorageManager: EventStorage {
                 logger.error("Warning: Could not decode event at \(url). Error: \(error)")
             }
         }
-        return events
+        // Sort events based on their storageId (which includes a timestamp).
```

Comment:
this comment does help big time. I still did not go to the tests yet, but usually I pick this things by looking at the tests too